app/src/main/res/values/strings.xml

```diff
@@ -2,7 +2,7 @@
     <string name="app_name">meteoapp</string>
 
     <!-- home page ui fixed values -->
-    <string name="next_5_days">Next 5 days</string>
+    <string name="next_5_days">NEXT 5 DAYS</string>
```

Comment:
You can leave the string as is, and then in app you could use .uppercase() if you need to be like NEXT 5 DAYS